### ISSUE TITLE
[Cleanup] Cleanup get_active_exchange_market

### ISSUE THREAD

**Issue by vic-en** [CoinAlpha/hummingbot#2554]
**Feature Suggestion**
The `get_active_exchange_market` was removed from the orderbook_data_source class because it was redundant after the `Discovery` strategy was deprecated. However, calls to that function still remain in the exchange files of most existing connector which should be removed as well.

**To-do:**
- Look through the exchange file of existing connectors and remove the function `get_active_exchange_market`. i.e in binance_exchange.pyx
```
    async def get_active_exchange_markets(self) -> pd.DataFrame:
        return await BinanceAPIOrderBookDataSource.get_active_exchange_markets()
```
- Remove the function from exchange base.
- Lastly, look through orderbook test files and remove test cases related to `get_active_exchange_markets`.

**Comment by mifeng** [comment]
check with backend team first

**Comment by vic-en** [comment]
After discussion with @paulfjacobs, we would only use `fetch_trading_pairs()` going forward.
`get_trading_pairs()` and `get_active_exchange_markets()` should be removed so we can have a cleaner code.
I'll make a PR but mark it as draft pending the time parrot is adjusted to use `fetch_trading_pairs()`.

**Comment by jason-hb** [comment]
Waiting on backend team

**Comment by DerivedClass** [comment]
I have discussed this with Victor and it looks like a simple change albeit in many places. I will raise with @martinkou in case there are any gotchas I am unaware of.

**Comment by dennisocana** [comment]
As discussed with @vic-en, we're still going to need get this task completed but not a high priority right now. We will need to coordinate with back-end team.

**Comment by dennisocana** [comment]
There were new connectors added after this story was put on hold. We should investigate first whether these connectors also use this function.

**Comment by dennisocana** [comment]
Updated the story with definition of done
